src/templates/rune.rs

```diff
@@ -7,6 +7,15 @@ pub(crate) struct RuneHtml {
   pub(crate) parent: Option<InscriptionId>,
 }
 
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+pub struct RuneJson {
+  pub entry: RuneEntry,
+  pub id: RuneId,
+  pub height: u32,
+  pub index: u16,
```

Comment:
```suggestion
```
This is contained in the RuneId so no need to add it here

Since RuneHtml and RuneJson are the same now you could just derive the same traits on RuneHtml and then type alias it to RuneJson. Something like this:

```rust
type RuneJson = RuneHtml;

#[derive(Boilerplate, Debug, PartialEq, Serialize, Deserialize)]
pub(crate) struct RuneHtml {
  pub(crate) entry: RuneEntry,
  pub(crate) id: RuneId,
  pub(crate) parent: Option<InscriptionId>,
}

```